docs/azure_arc_jumpstart/azure_arc_data/day2/aks/aks_mssql_dr/_index.md

```diff
@@ -0,0 +1,346 @@
+
+  - _Azure Arc Data Controller_ - The data controllers that are now deployed on the Kubernetes clusters.
+
+  - _Azure Arc-enabled SQL Managed Instance_ - The SQL Managed Instances that are now deployed on the Kubernetes clusters.
+
+    ![Screenshot showing additional Azure resources in the resource group](./32.png)
+
+- As part of the automation, Azure Data Studio is installed along with the _Azure Data CLI_, _Azure CLI_, _Azure Arc_ and the _PostgreSQL_ extensions. Using the Desktop shortcut created for you, open Azure Data Studio and click the Extensions settings to see the installed extensions.
+
+  ![Screenshot showing Azure Data Studio shortcut](./33.png)
+
+  ![Screenshot showing Azure Data Studio extensions](./34.png)
+
+- Additionally, the SQL Managed Instances connections will be configured automatically for you. As mentioned, the sample _AdventureWorks_ database was restored as part of the automation on the primary instance.
+
+  ![Screenshot showing Azure Data Studio SQL MI connection](./35.png)
+
+## Cluster extensions
+
+In this scenario, two Azure Arc-enabled Kubernetes cluster extensions were installed:
+
+- _azuremonitor-containers_ - The Azure Monitor Container Insights cluster extension. To learn more about it, you can check our Jumpstart ["Integrate Azure Monitor for Containers with GKE as an Azure Arc Connected Cluster using Kubernetes extensions](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/day2/gke/gke_monitor_extension/) scenario.
+
+- _arc-data-services_ - The Azure Arc-enabled data services cluster extension that was used throughout this scenario in order to deploy the data services infrastructure.
+
+In order to view these cluster extensions, click on the Azure Arc-enabled Kubernetes resource Extensions settings.
+
+![Screenshot showing the Azure Arc-enabled Kubernetes cluster extensions settings](./36.png)
+
+![Screenshot showing the Azure Arc-enabled Kubernetes installed extensions](./37.png)
+
+## Disaster recovery with SQL distributed availability groups
+
+Azure Arc-enabled SQL Managed Instance is deployed on Kubernetes as a containerized application and uses kubernetes constructs such as stateful sets and persistent storage to provide built-in health monitoring, failure detection, and failover mechanisms to maintain service health. For increased reliability, you can also configure disaster recovery between multiple Azure Arc-enabled SQL Managed Instances to be able to failover to another instance on another Kubernetes cluster. Disaster recovery is supported for both General Purpose and Business Critical tiers.
+
+## Operations
+
+### Azure Arc-enabled SQL Managed Instance distributed availability group validation
+
+- To be able to failover to a different cluster, a distributed availability group is created by the automation flow that spans the primary and secondary cluster. This can be validated by running the following commands:
```

Comment:
```suggestion
- To be able to failover to a different cluster, a distributed availability group is created by the automation flow that spans the primary and secondary clusters. This can be validated by running the following commands:
```